Scientific_workflows/WIT_Metrics/wit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p><code>!pip install --user git+git://github.com/GeoscienceAustralia/wit_tooling</code> fails on Sandbox due to the <code>--user</code>, but in principle works fine if you drop that flag. Change <code>git://</code>  to <code>https://</code>  which gets it working in Sandbox (I think because of whitelisting).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
This line doesn't work on sandbox, none of `dumping data from database` part would work on the sandbox. I should have been more specific about this.

Reply:
> I've added some suggested improvements. I can't run the notebook without the example files I mentioned, so if you add those, I can re-review and run the notebook!

the example files uploaded, can you run the notebook please?